src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -294,6 +294,22 @@ public ConfidentialClientApplicationBuilder WithCacheSynchronization(bool enable
             return this;
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="AppTokenProviderDelegate"></param>
+        /// <returns></returns>
+        public ConfidentialClientApplicationBuilder WithAppTokenProvider(Func<AppTokenProviderParameters, Task<ExternalTokenResult>> AppTokenProviderDelegate)

```

Comment:
naming `appTokenProviderDelegate`

Am curious - is VS not warning you about the naming violations?

Reply:
no, I dont see it

Reply:
Something for an OCE to fix :).